doc/api/api_changes/2017-12-28-RH.rst

```diff
@@ -0,0 +1,4 @@
+New rcParam for Legend Title Font Size
+``````````````````````````````````````
+
+The introduction of a new rcParam for legend title font size (``legend.titlesize``) deprecates the previous usage of the ``font.size`` rcParam for modification of the legend title font size.
```

Comment:
I think you mean "deprecate" -> "replaces".  To me, at least, "deprecation" means you will still let it work, but it will stop working at some point.  You aren't deprecating, you are doing a hard replace.  

I think this should explicitly state that old code where `font.size` was changed will now also need to change `legend.titlesize` to get the same effect, and that the default is `medium` (as is `font.size`).